plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -220,6 +252,19 @@ public static IcebergFileFormat getFileFormat(Map<String, Object> tablePropertie
         return (IcebergFileFormat) tableProperties.get(FILE_FORMAT_PROPERTY);
     }
 
+    public static Optional<HiveCompressionCodec> getHiveCompressionCodec(Map<String, Object> inputProperties)
+    {
+        return Optional.ofNullable((HiveCompressionOption) inputProperties.get(WRITE_COMPRESSION))
+                .map(HiveCompressionCodecs::toCompressionCodec);
+    }
+
+    public static OptionalInt getCompressionLevel(Map<String, Object> tableProperties)
+    {
+        return Optional.ofNullable((Integer) tableProperties.get(COMPRESSION_LEVEL))
+                .map(OptionalInt::of)
+                .orElseGet(OptionalInt::empty);
```

Comment:
Nit: no need for a lambda here, just `orElse(OptionalInt.empty())`

Reply:
done